packages/jsii-diff/bin/jsii-diff.ts

```diff
@@ -81,7 +77,7 @@ async function loadPackageNameFromAssembly(): Promise<string> {
   if (!await fs.pathExists(JSII_ASSEMBLY_FILE)) {
     throw new Error(`No NPM package name given and no ${JSII_ASSEMBLY_FILE} file in the current directory. Please specify a package name.`);
   }
-  const module = await fs.readJSON(JSII_ASSEMBLY_FILE, { encoding: 'utf-8' }) as spec.Assembly;
+  const module = spec.validateAssembly(await fs.readJSON(JSII_ASSEMBLY_FILE, { encoding: 'utf-8' }));
   if (!module.name) { throw new Error(`Could not find package in ${JSII_ASSEMBLY_FILE}`); }
```

Comment:
I don't reckon this is possible anymore, not that you've run through `validateAssembly` :)